src/utils/hooks/useOnClickOutside.js

```diff
@@ -0,0 +1,33 @@
+/* globals document */
+// Adopted from https://usehooks.com/useOnClickOutside/
```

Comment:
I lifted this from an external source and wasn't able to find a great way to test it. Let me know if this is a stopper for merge.

Reply:
All good